src/AnimationLC.cpp

```diff
@@ -9,26 +9,40 @@
 #include <OgreEntity.h>
 #include <json.h>
 
+#include <iostream>
+
```

Comment:
If the other suggestion lands, make sure to land this one as well to avoid unused includes.

🧹
```suggestion
```